src/redux/books/booksSlice.js

```diff
@@ -1,7 +1,24 @@
+/* eslint-disable no-param-reassign */
 import { createSlice } from '@reduxjs/toolkit';
```

Comment:
* I can see you have created a category slice Kindly use this slice in the category component **HINT** 💭 you could render `Under Construction` when the check status button is clicked 🙏

Reply:
resolved! ✔️